Assets/Editor/BuildTiltBrush.cs

```diff
@@ -936,12 +943,12 @@ public void Dispose()
     class TempSetBundleVersion : IDisposable
     {
         string m_prevBundleVersion;
-        public TempSetBundleVersion(string configVersionNumber, string stamp)
+        public TempSetBundleVersion(BuildTarget target, string configVersionNumber, string stamp)
         {
             m_prevBundleVersion = PlayerSettings.bundleVersion;
             // https://stackoverflow.com/a/9741724/194921 for more on the meaning/format of this string
             PlayerSettings.bundleVersion = configVersionNumber;
-            if (!string.IsNullOrEmpty(stamp))
+            if (!string.IsNullOrEmpty(stamp) && target != BuildTarget.iOS)
```

Comment:
What happens if we want to install an iOS build from a branch? Does it overwrite the main build?

Reply:
I just installed this PR for testing and it installed separately to my local build, so I guess not? anyways, this was causing a build failure related to having brackets in the name

Reply:
Can you add a comment that we can't use brackets (to be clear, I assume you mean what I'd call parentheses -- these things: `()`, right).

This primarily happens when we build from the GUI, where the stamp is `-(menuitem)`. 

Can we just replace those characters instead?

Reply:
Yep parentheses :D yes I suppose so, I'll change it to just be `-menuitem`

Reply:
Found the actual error: `UnityException: iOS Version has not been set up correctly, it must consist only of '.'s and numbers, must begin and end with a number and be no longer than 18 characters`